src/client/components/EmployersQuestions/index.js

```diff
@@ -182,10 +184,12 @@ function EmployerQuestions(props) {
       {renderTextInput("totalHoursWorked", {
         className: "col-md-2",
         pattern: "\\d+[.]?\\d*",
+        maxLength: 3,
```

Comment:
Do we even need to allow fractional hours? Should we update the pattern to just \d+?

Reply:
Oh good point! The max should probably be 6 characters if we're allowing decimals. @nguyen-tina do you have a preference here? I'll also check the DB to see if anybody has submitted decimals yet.

Reply:
Yes, we should allow for decimals and extra characters to get fractional hours! I agree with changing to max 6 characters

Reply:
Sounds good! Updating now!